overlore/sqlite/npc_db.py

```diff
@@ -60,43 +59,31 @@ def init(self, path: str = "./npc_spawn.db") -> NpcDatabase:
         )
         return self
 
-    def get_all_npc_spawn(self) -> list[Npc]:
-        return self.execute_query("SELECT * FROM npc_spawn", ())
-
-    def get_all_npc_info(self) -> list[Npc]:
-        return self.execute_query("SELECT * FROM npc_info", ())
-
-    def insert_npc_spawn(self, realm_entity_id: int, npc: Npc) -> int:
-        query = (
+    def insert_npc_spawn(self, realm_entity_id: int, npc: NpcProfile) -> int:
```

Comment:
Can we have a more explicit name? Don't understand what is an `npc_spawn`

Reply:
Changed it to npc_profile for clarity